suse_macros.in

```diff
@@ -222,18 +222,11 @@
 
 # for %post
 %install_info(:-:) \
-    if test -x /sbin/install-info ; then \
-	/sbin/install-info %{**} || : \
-    fi ;
+   # legacy install_info macro used, please fix the package
```

Comment:
Wouldn't it be better to use `%{error:}` or `%{warn:}` here? https://rpm.org/user_doc/macros.html#builtin-macros

Reply:
I was actually thinking of writing an rpmlint check that triggers on that comment in binary rpms. %warn wouldn't leave traces in the binary rpm so nothing we can look for.
Grepping the spec file on the other hand would be too naive as the macro could be wrapped in %if ...

Reply:
Using `%error` would stop the package from being able to use the macro when it's not supposed to be used.

Reply:
that's not what we normally do to deprecate macros. Would force packagers to wrap in %if %suse_version themselves which is annoying.

Reply:
I'd rather expand to %{nil} and forget about checks. spec_cleaner could remove it then.

Reply:
That works too.